README.md

```diff
@@ -1,6 +1,6 @@
 ![containerd banner](https://raw.githubusercontent.com/cncf/artwork/master/projects/containerd/horizontal/color/containerd-horizontal-color.png)
 
-[![GoDoc](https://godoc.org/github.com/containerd/containerd?status.svg)](https://godoc.org/github.com/containerd/containerd)
+[![PkgGoDev](https://pkg.go.dev/badge/github.com/containerd/containerd)](https://pkg.go.dev/github.com/containerd/containerd)
```

Comment:
Looks like there's two possible landing pages;

- https://pkg.go.dev/mod/github.com/containerd/containerd (shows the readme and link to repo)
- https://pkg.go.dev/github.com/containerd/containerd (only the go module docs)

Wondering which one is more appropriate?

Reply:
The latter one.

Reply:
agree.. the latter one.. more tabs... users coming from our repo readme don't need to land on another copy of the readme.. and both landing pages include link backs to the repo..

Reply:
@thaJeztah I generated it from https://pkg.go.dev/badge/ and I think the later one is better as Mike said. :p

Reply:
> users coming from our repo readme don't need to land on another copy of the readme

Ah, yes, that's fair.

I got somewhat confused, because the first link showed "overview" as first tab, so felt more like an "entry point" to the project.  (the second one shows the same info, but now as _second_ tab 🤷‍♂️). I guess my confusion was because the old godoc.org also shows the subpackages/directories on the landing page, whereas the new layout only shows the root package, and [hides subdirectories/packages in the third tab](https://pkg.go.dev/github.com/containerd/containerd?tab=subdirectories)